src/test/scala/mesosphere/marathon/api/TestAuthFixture.scala

```diff
@@ -16,8 +16,8 @@ class TestAuthFixture() extends Mockito {
 
   var identity: Identity = new Identity {}
 
-  var authenticated: Boolean = true
-  var authorized: Boolean = true
+  @volatile var authenticated: Boolean = true
+  @volatile var authorized: Boolean = true
```

Comment:
What is the reasoning behind this change?

Reply:
possibly concurrent write/read access to those vars